### PR TITLE
vita3k: Fix grammar errors

### PR DESCRIPTION
**Pull request by EXtremeExploit**

### CODE REVIEW

File: vita3k/gui/src/themes.cpp [hunk 1]
```diff
@@ -214,17 +214,17 @@ bool init_theme(GuiState &gui, HostState &host, const std::string &content_id) {
                 theme_icon_name["NPXS10026"] = theme.child("HomeProperty").child("m_hostCollabo").child("m_iconFilePath").text().as_string();
 
             // Get Bar Color
-            if (!theme.child("InfomationBarProperty").child("m_barColor").empty())
-                bar_param["barColor"] = theme.child("InfomationBarProperty").child("m_barColor").text().as_string();
-            if (!theme.child("InfomationBarProperty").child("m_indicatorColor").empty())
-                bar_param["indicatorColor"] = theme.child("InfomationBarProperty").child("m_indicatorColor").text().as_string();
+            if (!theme.child("InformationBarProperty").child("m_barColor").empty())
```
Comment (Zangetsu38) [inline vita3k/gui/src/themes.cpp:217]: revert this file, is not wrong child name is really this
><InfomationBarProperty>

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]

File: vita3k/gui/src/themes.cpp [hunk 2]
```diff
@@ -406,36 +406,36 @@ void init_themes(GuiState &gui, HostState &host) {
             pugi::xml_document doc;
 
             if (doc.load_file(theme_path_xml.c_str())) {
-                const auto infomation = doc.child("theme").child("InfomationProperty");
+                const auto information = doc.child("theme").child("InfomationProperty");
```
Comment (Zangetsu38) [inline vita3k/gui/src/themes.cpp:409]: like i said, revert this file, i want keep const value named like child ^^

Comment (EXtremeExploit) [reply vita3k/gui/src/themes.cpp:409]: sony made mistakes, we aren't following them, right? right?

Comment (Zangetsu38) [reply vita3k/gui/src/themes.cpp:409]: i am no sur is error seriously haha and i just want keep same name for more easy follow

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]
revert theme file, and rename commit to "project: fix grammar" exemple ^^ never forget add what your target before explain

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**